src/main/java/org/variantsync/diffdetective/variation/diff/LineDiffer.java

```diff
@@ -0,0 +1,62 @@
+package org.variantsync.diffdetective.variation.diff;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+import org.variantsync.diffdetective.diff.result.DiffParseException;
+import org.variantsync.diffdetective.util.CompositeSource;
+import org.variantsync.diffdetective.util.Source;
+import org.variantsync.diffdetective.variation.DiffLinesLabel;
+import org.variantsync.diffdetective.variation.diff.parse.VariationDiffParseOptions;
+import org.variantsync.diffdetective.variation.diff.parse.VariationDiffParser;
+
+/**
+ * A {@link Differ} that first creates a line diff and
```

Comment:
I would prefer the name `TextBasedDiffer` over `LineDiffer`. At least "text-based differencing" is the term used in research, our papers, and my thesis. What do you think?